code/datums/mob_stats.dm

```diff
@@ -68,17 +68,26 @@
 	var/datum/stat/S = stat_list[statName]
 	S.setValue(Value)
 
+/datum/stat_holder/proc/change_Stat_cap(statName, amount)
```

Comment:
```suggestion
/datum/stat_holder/proc/add_Stat_cap(statName, amount)
```
A proc named "change stat cap" is unclear whether it adds the value or sets a new one.